pkg/destroy/aws/suppresswarning.go

```diff
@@ -0,0 +1,25 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	errorTracker = map[string]time.Time{}
```

Comment:
personally not a fan of this global variable.. maybe just pass it along to the function..?

Reply:
Moved to a struct.